downloads/tests/test_views.py

```diff
@@ -554,3 +554,46 @@ def test_filter_release_file_delete_by_release(self):
             headers={"authorization": self.Authorization}
         )
         self.assertEqual(response.status_code, 405)
+
+class ReleaseFeedTests(BaseDownloadTests):
+    """Tests for the downloads/feed.rss endpoint.
+
+    Content is ensured via setUp in BaseDownloadTests.
+    """
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.url = reverse("downloads:feed")
```

Comment:
This can be defined as a class attribute, which is a bit cleaner.

```
class ReleaseFeedTests(BaseDownloadTests):
    url = reverse("downloads:feed")
    ...
```

and then there's no need to define a setUp.

Reply:
should be done in [075ef04](https://github.com/python/pythondotorg/pull/2569/commits/075ef047b6463570bce1c85d4f87946a4d28e6ec)